libs/model/src/aggregates/user/signIn/privy.ts

```diff
@@ -109,7 +109,6 @@ export async function signInPrivy({
     // TODO: existing linkedAccounts array may be sufficient to get the verifiedUserInfo and avoid
     //  this extra call to Privy
     const fullPrivyUser = await getPrivyUserById(privyUser.id);
-    if (!payload.privy.ssoOAuthToken) throw new Error('Missing OAuth token');
```

Comment:
Rather than removing it, it should be checked that it is provided for whatever subset of providers it is required for (or vice versa).